### ISSUE TITLE
Bug in Text2Text Annotation

### ISSUE THREAD

**Issue by hkjeon13** [recognai/rubrix#1816]
**Describe the bug**
In Annotation works with the "Text2Text" module, I can't type annotation of more than 1 character.
After the annotation area got one character, the input area focused out so I click again the area once again(type 1 char > click > type 1 char > click > ...).
So I can type annotation text only with the "copy & paste" method with other streams.
These symptoms have occurred in every annotation area. 

**To Reproduce**
1. Run docker environment
```
$docker run -d --name es-for-argilla -p 9200:9200 -p 9300:9300 -e "ES_JAVA_OPTS=-Xms512m -Xmx512m" -e "discovery.type=single-node" docker.elastic.co/elasticsearch/elasticsearch-oss:7.10.2

```
 
2. Run argilla
```
$ python -m argilla
```
3. Build the script for register and run it
```
import argilla as rg
import pandas as pd
from datasets import Dataset

df = pd.read_excel("sample.xlsx")
df = df[['labels']]
df.columns = ["text"]
df = df.dropna()
dataset = Dataset.from_pandas(df)
dataset_rg = rg.read_datasets(dataset, task="Text2Text")

rg.log(dataset_rg, name="spell-correction")
```
4. Enter the local address "http://0.0.0.0:6900"
5. Click the "annotate" tab.
6. Enter the annotion in the text area.
7. You can see the error focused out after 1 character 

**Expected behavior**
After typing 1 character, the point has been focused out, and I can't type anymore(For the continuation, I must click again).
**Screenshots**
![](https://i.ibb.co/ZSfBFHn/2022-11-02-11-32-24.png)

**Environment (please complete the following information):**
 - OS [e.g. iOS]: MacOS Ventura
 - Browser [e.g. chrome, safari]: chrome
 - Argilla Version [e.g. 1.0.0]: 1.0.0
 - ElasticSearch Version [e.g. 7.10.2]: 7.10.2
 - Docker Image (optional) [e.g. argilla:v1.0.0]: argilla:v1.0.0

**Additional context**

**Comment by leiyre** [comment]
Hello @hkjeon13, thank you very much for reporting the bug. We are working on it and hope to have it fixed very soon.

**Comment by hkjeon13** [comment]
@leiyre thank you for your concerns!

**Comment by leiyre** [comment]
Hello @hkjeon13, the issue has already been fixed, you can update to version `1.0.1`. 
https://github.com/argilla-io/argilla/releases/tag/v1.0.1, 
Thank you
